### PR TITLE
UG Edit, Invalid Input standardisation and PPP

### PR DESCRIPTION
**Pull request by firwer**

### CODE REVIEW

File: docs/team/pohwp.md
Comment (ngkaiwen123) [inline docs/team/pohwp.md:1]: I think the name of the md file needs to be renamed to your github username for the website to detect your PPP draft?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -393,25 +407,25 @@ Format: `sell upc/[UPC] qty/[Quantity]`
...
 **REQUIRED** Parameters:
 
-* The `upc/` parameter whereby `[UPC]` refers to the identification number assigned to the item at the point 
+* The `[UPC]` field refers to the identification number assigned to the item at the point 
 of **initial addition** of the item. `[UPC]` has to be valid, that is, it **EXISTS** in the database, and has to be 
 a **POSITIVE NUMBER** and **NOT EMPTY**. Do **NOT** include `[` and `]` in the input.
-* The `qty/` parameter whereby `[Quantity]` refers to the amount of stock to be **DEDUCTED** from the current 
+* The `[Quantity]` field refers to the amount of stock to be **DEDUCTED** from the current 
 stock levels recorded. `[Quantity]` input **SHOULD NOT** be **EMPTY**,a **NEGATIVE INTEGER**, **ZERO** or a **STRING**. 
 Do **NOT** include `[` and `]` in the input.
```
Comment (ngkaiwen123) [inline docs/UserGuide.md:415]: For both the sell and restock ug there might be merge conflicts, I have deleted the wordy parts of the upc and quantity and shifted it below into the form of a warning(!>)

Comment (firwer) [reply docs/UserGuide.md:415]: can can, ill resolve it accordingly after you have merged

File: docs/team/pohwp.md
Comment (firwer) [reply docs/team/pohwp.md:1]: Ohhh okay i didn't know ill make the changes

**Pull request merged**